index.php

```diff
@@ -519,19 +525,19 @@ function setWikiLayer(visible){
 
             function setWikiProtocol(){
               setCookie("WikipediaLayerThumbs", wikipediaThumbs);
-              var displayThumbs = wikipediaThumbs ? 'yes' : 'no';
-              var iconsProtocol = new OpenLayers.Protocol.HTTP({
-                        url: 'api/proxy-wikipedia.php?',
-                  params: {
-                      'LANG' : language,
-                      'thumbs' : displayThumbs
-                  },
-                  format: new OpenLayers.Format.KML({
-                      extractStyles: true,
-                      extractAttributes: true
-                  })
-              });
-              layer_wikipedia.protocol = iconsProtocol;
+            //   var displayThumbs = wikipediaThumbs ? 'yes' : 'no';
+            //   var iconsProtocol = new OpenLayers.Protocol.HTTP({
+            //             url: 'api/proxy-wikipedia.php?',
+            //       params: {
+            //           'LANG' : language,
+            //           'thumbs' : displayThumbs
+            //       },
+            //       format: new OpenLayers.Format.KML({
+            //           extractStyles: true,
+            //           extractAttributes: true
+            //       })
+            //   });
```

Comment:
Did you move all the commented functionality elsewhere or is this still a to-do?

Reply:
This commented code must be removed. The update  of the layer when `wikipediaThumbs` variable changes is implemented is in `layer_wikipedia` layer definition and triggered via the next line "`layer_wikipedia.getSource().refresh()`